### PR TITLE
Shared values should share same localised strings

### PR DESCRIPTION
**Pull request by raamSoftwire**
## Context

There are a lot of strings that are unlikely to change and are unnecessarily duplicated in the strings files.

## Changes proposed in this pull request

We analyzed all the strings and counted the number of duplications to find the best candidates for refactoring, you can find the results here, [locales.xlsx](https://github.com/DFE-Digital/apply-for-teacher-training/files/5829583/locales.xlsx). This PR currently refactors
- "Continue"
- "Save and continue" 

I'm not sure how far down that list to go, but this feels like a good start.

## Guidance to review

- Review commit by commit

## Link to Trello card

https://trello.com/c/S1sZ3qdy/366-low-shared-values-should-share-same-localised-strings

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Review by george-bit — APPROVED** [review]
LGTM @raamSoftwire, Much neater! (Appreciate the structured commits too, easy to review)

File: spec/support/test_helpers/candidate_helper.rb [hunk 1]
```diff
@@ -151,41 +151,41 @@ def candidate_fills_in_personal_details
...
     check 'British'
     check 'Citizen of a different country'
     within('#candidate-interface-nationalities-form-other-nationality1-field') do
       select 'American'
     end
-    click_button t('complete_form_button', scope: scope)
+    click_button t('application_form.complete_form_button')
 
     check t('application_form.completed_checkbox')
     click_button 'Continue'
```
Comment (paulrobertlloyd) [inline spec/support/test_helpers/candidate_helper.rb:164]: Should this be localised?

**Comment by paulrobertlloyd** [comment]
Two things:

* Should these strings live in a more global location. Currently they exist per localisation
* Have caught any occurrences of unlocalised strings that use these values?

**Comment by davidgisbey** [comment]
Good work Raam. I agree with Paul that something like 'Continue' should live in a top-level file that can be shared across the interfaces. 

This is way neater though and is looking great 👍

File: spec/support/test_helpers/candidate_helper.rb (hunk 1 above)
Comment (raamSoftwire) [reply spec/support/test_helpers/candidate_helper.rb]: Done

**Comment by raamSoftwire** [comment]
> Two things:
> 
> * Should these strings live in a more global location. Currently they exist per localisation
> * Have you checked for any occurrences of unlocalised strings that use these same values?

I've done both of these now.

**Review by paulrobertlloyd — APPROVED** [review]
❤️ it!

**Pull request merged**